core/lib/basic_types/src/protocol_version.rs

```diff
@@ -121,6 +121,12 @@ impl ProtocolVersionId {
     pub fn is_pre_1_4_2(&self) -> bool {
         self < &ProtocolVersionId::Version21
     }
+
+    pub fn is_pre_1_5_0(&self) -> bool {
+        // In the current codebase all the protocol versions are pre-1.5.0.
+        // This method will be updated once the v1.5.0 is added to the server
+        true
+    }
```

Comment:
There is already `is_pre_shared_bridge` method, it should be merged with `is_pre_1_5_0` (I don't think it should be done but we shouldn't forget about it later)

Reply:
@perekopskiy thank you flagging this. Recently @mm-zk said that he is not 100% sure that 1.5.0 will go together with shared bridge, though 99% that these will, so let's probably merge those as part of the Kalman's branch in case these upgrades are mergeed